cmd/geth/main.go

```diff
@@ -301,37 +317,32 @@ func startNode(ctx *cli.Context, stack *node.Node) {
 		utils.Fatalf("Unable to attach to node: %v", err)
 	}
 
+	// Perform addt'l unlock steps depending on whether this is a
+	// maker or voter.  Observers are handled by first unlock above.
 	var (
 		voteKey      *ecdsa.PrivateKey
 		blockVoteKey *ecdsa.PrivateKey
 	)
-
-	if addr := ctx.GlobalString(utils.VoteAccountFlag.Name); addr != "" {
-		addr = strings.TrimSpace(addr)
-		var passwd []string
-		if ctx.GlobalIsSet(utils.VoteAccountPasswordFlag.Name) {
-			passwd = append(passwd, ctx.GlobalString(utils.VoteAccountPasswordFlag.Name))
-		}
-
-		unlockAccount(ctx, accman, addr, 0, passwd)
-		// unlockAccounts fatals in case the account could not be unlocked
-		voteKey, err = accman.Key(common.HexToAddress(addr))
-		if err != nil {
-			utils.Fatalf("Unable to unlock vote key: %v", err)
-		}
+	usingVoterAcct := ctx.GlobalIsSet(utils.VoteAccountFlag.Name)
+	usingBlockMakerAcct := ctx.GlobalIsSet(utils.VoteBlockMakerAccountFlag.Name)
+	if len(accounts) == 0 && !usingVoterAcct && !usingBlockMakerAcct {
+		utils.Fatalf("Was not provided an `unlock`, `voteaccount`, or `blockmakeraccount` flag, cannot launch.")
```

Comment:
Yeah, this is what I meant about my assumption that all nodes will be either observers, voters, or makers, respectively.  If a node isn't unlocking any accounts on startup, I'm explicitly throwing an error.

Is that not valid behavior w/ how this will be run?